src/CoreBundle/private/modules/MTHeader/views/standard.view.php

```diff
@@ -2,55 +2,91 @@
+        <button type="button" class="navbar-toggler sidebar-toggler d-lg-none" data-toggle="sidebar-show">
             <span class="navbar-toggler-icon"></span>
         </button>
-
-        <?php
-        if (isset($data['oUser'])) {
-            ?>
-        <div class="collapse navbar-collapse" id="cmsTopNavbar">
-            <?php
-            if (isset($data['check_messages'])) {
-                ?>
-                <div class="btn-group">
-                    <button type="button" class="btn btn-sm dropdown-toggle navbar-btn btn-secondary" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" role="button">
-                        <span class="glyphicon glyphicon-warning-sign"></span> <?=TGlobal::OutHTML($translator->trans('chameleon_system_core.admin_message.button_title')); ?>
-                    </button>
-                    <div class="dropdown-menu" aria-labelledby="adminMessagesDropDown" style="min-width: 370px;">
-                        <div class="row">
-                            <div class="col-lg-12">
-                                <ul>
-                                    <?php
-                                    foreach ($data['check_messages'] as $message) {
-                                        echo '<li class="dropdown-item">'.$message.'</li>';
-                                    } ?>
-                                </ul>
-                            </div>
-                        </div>
-                    </div>
-                </div>
+        <a href="<?=PATH_CMS_CONTROLLER; ?>?_rmhist=true&_histid=0" class="navbar-brand d-sm-down-none">
+            <img src="<?= TGlobal::OutHTML($sLogoURL) ?>" alt="" />
+        </a>
+        <button type="button" class="navbar-toggler sidebar-toggler d-md-down-none" data-toggle="sidebar-lg-show">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+            <ul class="nav navbar-nav">
                 <?php
-            }
-            ?>
-            <ul class="navbar-nav mr-auto">
+                if (true === isset($data['check_messages'])) {
+                    ?>
+                    <li class="nav-item px2 dropdown">
+                        <a
+                                class="nav-link dropdown-toggle text-danger"
+                                data-toggle="dropdown"
+                                href="#"
+                                role="button"
+                                aria-haspopup="true"
+                                aria-expanded="false"
+                        >
+                            <i class="fas fa-exclamation-triangle"></i>
+                            <span class="d-md-down-none">
+                                <?= TGlobal::OutHTML($translator->trans('chameleon_system_core.admin_message.button_title')) ?>
+                            </span>
+                        </a>
+                        <div class="dropdown-menu dropdown-menu-left">
```

Comment:
messages are a "dropdown-menu"?

Reply:
Yes, so an alert button is displayed, and the error messages can be viewed in the dropdown "menu" after clicking the button.